apps/arweave/src/ar.erl

```diff
@@ -311,7 +311,8 @@ show_help() ->
 			{"is_pool_client", "Configure the node as a pool client. The node may be an "
 					"exit peer in the coordinated mining setup or a standalone node."},
 			{"pool_api_key", "API key for the requests to the pool."},
-			{"pool_server_address", "The pool address"}
+			{"pool_server_address", "The pool address"},
+			{"pool_worker_name", "(optional) The pool worker name"}
```

Comment:
@vird fine if you insist on keeping the naming inconsistent, then I think you need to explain the flag better. Right now it is confusing to me, likely for users, and for new developers to the codebase. Nowhere else in our documentation do we refer to a pool worker. We refer to pool clients and pool servers - so your documentation should clarify what a pool worker name is and how it different from the pool client and pool server

Reply:
tl;dr word client has no sence in terms of pool. So it was accidentally introduced in our code and now we are relying on it as standard

I propose to remove word client at all (in future, not now)
word client is only used in is_pool_client and in cm_peer description
Also in vdf_client_peer, but that's different

is_pool_client was introduced same time as is_pool_server to distinguish those 2 modes
So currently client word is totally redundant

How it should work. No matter CM + pool or CM + solomine. CM peers should be configured same way (currently it's maybe not)
So the only place when is_pool_client can be present it's exit peer, so in this case we already have pool_server_address

is_pool_server maybe keep as is

Reply:
👍 

logged: https://github.com/ArweaveTeam/arweave-dev/issues/464